test/e2e/specs/editor/various/patterns.spec.js

```diff
@@ -252,4 +252,49 @@ test.describe( 'Synced pattern', () => {
 			},
 		] );
 	} );
+
+	// Check for regressions of https://github.com/WordPress/gutenberg/issues/26421.
+	test( 'allows conversion back to blocks when the reusable block has unsaved edits', async ( {
```

Comment:
The original test tested when there are unsaved edits to the reusable block entity, which seems to be lost now in this new test, but I think is still a valuable test.

The new test is also valuable, but maybe it's worth also keeping the original and restoring it at some point rather than deleting it (it would have to be changed to use the 'Edit original' button to update the entity).

Given this new test is for overrides, is it worth moving it to the `pattern-overrides` file? Suggestion for a new test title:
```suggestion
	test( 'retains override values when converting a pattern block to regular blocks', async ( {
```

Reply:
Hmm, right! Makes sense!

Reply:
Updated in [fe5aa44](https://github.com/WordPress/gutenberg/pull/58164/commits/fe5aa44c38730f1164f6148e47e821e7b3b63858).